cmake/AwsPrebuildDependency.cmake

```
@@ -75,3 +92,52 @@ function(aws_prebuild_dependency)
         DESTINATION ${CMAKE_INSTALL_PREFIX}
     )
 endfunction()
+
+# Get list of optional or platform-specific variables that may affect build process.
+function(aws_get_variables_for_prebuild_dependency AWS_CMAKE_PREBUILD_ARGS)
+    set(variables "")
+    set(variablesToIgnore CMAKE_INSTALL_PREFIX)
+
+    # The CMake variables below were chosen for Linux, BSD, and Android platforms. If you want to use the prebuild logic
+    # on other platforms, the chances are you have to handle additional variables (like CMAKE_OSX_SYSROOT). Refer to
+    # https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html to update the list of handled variables, and
+    # then you can enable a new platform here.
+    if (NOT UNIX OR APPLE)
```

trivial: add parens to make this more clear
```suggestion
    if ((NOT UNIX) OR APPLE)
```